tripy/tripy/common/utils.py

```diff
@@ -138,6 +69,10 @@ def convert_list_to_array(values: List[Any], dtype: str) -> bytes:
     return array.array(TYPE_TO_FORMAT[dtype], values)
 
 
+def has_no_contents(data: Sequence) -> bool:
```

Comment:
nit: could we just call this `is_empty`?

Reply:
Updated.